packages/utils/src/build-context.js

```diff
@@ -216,7 +231,10 @@ function getAvatarUrl(hash = 'HEAD') {
     encoding: 'utf8',
   });
   if (result.status !== 0) {
-    throw new Error('Unable to determine commit email with `git log --format=%aE -n 1`');
+    throw new Error(
+      "Couldn't get commit author's avatar url because application was unable to determine commit email with `git log --format=%aE -n 1`. " +
```

Comment:
```suggestion
      "Unable to determine commit author's avatar URL because `git log --format=%aE -n 1` failed to provide author's email. " +
```

feels slightly more inline with the way the others are worded, but not a big deal

Reply:
The extra push to fix should also retry the tests which look like a flaky failure of puppeteer :)

Reply:
This was marked as resolved but no explanation. Do you prefer to keep the existing wording? If so I'm curious why :)

Reply:
> This was marked as resolved but no explanation. Do you prefer to keep the existing wording? If so I'm curious why :)

Sorry, forgot to click `Commit suggestion`. Of course if the wording you proposed better suits the pattern of the app then sure :)